packages/demo/src/components/examples/FooterExample.tsx

```diff
@@ -31,12 +44,28 @@ const InputWithDirty = withDirtyInputHOC(InputConnected);
 const StickyFooterWithDirty = withDirtyStickyFooterHOC(StickyFooter);
 const INPUT_ID = 'input-for-something';
 
+const ContentWithStickyFooter: React.FunctionComponent<Partial<IWithDirty> & ISplitLayoutProps> = _.compose(
+    withDirty({

```

Comment:
The example could pass the showDirty function from the props instead of the config, since it is the best way of doing things.

```tsx
const ContentWithStickyFooter: React.FunctionComponent<Partial<IWithDirty> & ISplitLayoutProps> 
    = _.compose(withDirty())(SplitLayout);

// when rendering it
<ContentWithStickyFooter
    id='FooterWithContent'
    leftChildren={/* whatever */}
    rightChildren={/* whatever */}
    showDirty={() => (
        <StickyFooterWithDirty className="sticky-footer-mod-header" validationIds={[INPUT_ID]}>
            <Button primary name="fake button" />
        </StickyFooterWithDirty>
    )}
    isDirty
/>
```